tests/schema.test.ts

```diff
@@ -9,7 +9,7 @@ const defaults = {
   immutableTypes: false,
   required: new Set<string>(),
   rawSchema: false,
-  version: 3,
+  version: 3
```

Comment:
Not blocking for this PR, but if you could use the local Prettier settings for this project that’d cut down on some unnecessary diff here